docs/LogGuide.md

```diff
@@ -2821,3 +2853,399 @@ Parsed Log Line Examples:
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+270|2024-03-02T13:14:37.0430000-08:00|4000F1D3|-2.2034|0002|0014|102.0539|118.1982|0.2136|4601ae28c0b481d8
+270|2024-03-02T13:18:30.2960000-08:00|4000F44E|2.8366|0002|0014|98.2391|101.9623|0.2136|2eed500a1505cb03
+270|2024-03-02T13:18:30.6070000-08:00|4000F44E|-2.5710|0002|0014|98.2391|101.9318|0.2136|51bc63077eb489f3
+
+Parsed Log Line Examples:
+[13:14:37.043] 270 10E:4000F1D3:-2.2034:0002:0014:102.0539:118.1982:0.2136
+[13:18:30.296] 270 10E:4000F44E:2.8366:0002:0014:98.2391:101.9623:0.2136
+[13:18:30.607] 270 10E:4000F44E:-2.5710:0002:0014:98.2391:101.9318:0.2136
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=ActorMove&lang=en-US) -->
+
+<a name="line271"></a>
+
+### Line 271 (0x10F): ActorSetPos
+
+An `ActorSetPos` packet is sent to instruct the game client to set the position of an actor
+with no interpolated movement (for example, in UWU, to set the location of the Ifrit clones).
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=ActorSetPos&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+271|[timestamp]|[id]|[heading]|[?]|[?]|[x]|[y]|[z]
+
+Parsed Log Line Structure:
+[timestamp] 271 10F:[id]:[heading]:[?]:[?]:[x]:[y]:[z]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>271)\|(?<timestamp>[^|]*)\|(?<id>[^|]*)\|(?<heading>[^|]*)\|(?:[^|]*\|){2}(?<x>[^|]*)\|(?<y>[^|]*)\|(?<z>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 271 (?<type>10F):(?<id>[^:]*):(?<heading>[^:]*)(?::[^:]*){2}:(?<x>[^:]*):(?<y>[^:]*):(?<z>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+271|2024-03-02T13:20:50.9620000-08:00|4000F3B7|-2.3563|00|00|116.2635|116.2635|0.0000|e3fa606a5d0b5d57
+271|2024-03-02T13:20:50.9620000-08:00|4000F3B5|-1.5709|00|00|107.0000|100.0000|0.0000|5630c8f4e2ffac77
+271|2024-03-02T13:20:50.9620000-08:00|4000F3BB|0.2617|00|00|97.4118|90.3407|0.0000|01d53a3800c6238f
+
+Parsed Log Line Examples:
+[13:20:50.962] 271 10F:4000F3B7:-2.3563:00:00:116.2635:116.2635:0.0000
+[13:20:50.962] 271 10F:4000F3B5:-1.5709:00:00:107.0000:100.0000:0.0000
+[13:20:50.962] 271 10F:4000F3BB:0.2617:00:00:97.4118:90.3407:0.0000
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=ActorSetPos&lang=en-US) -->
+
+<a name="line272"></a>
+
+### Line 272 (0x110): SpawnNpcExtra
+
+This line contains certain data from `NpcSpawn` packets not otherwise made available
+through other log lines.
+
+The `tetherId` field is the same as the `id` field used in
+[NetworkTether lines](#line-35-0x23-networktether) and corresponds to an id in the
+[Channeling table](https://github.com/xivapi/ffxiv-datamining/blob/master/csv/Channeling.csv).
+
+The `animationState` field reflects the initial animation state of the actor
+at time it is spawned, and corresponds to the
+[BNpcState table](https://github.com/xivapi/ffxiv-datamining/blob/master/csv/BNpcState.csv).
```

Comment:
Might be worth mentioning the nuance here of "if the actor spawns with a tether or animation state value, there will not be a corresponding `NetworkTether` or `ActorControlExtra` line emitted to indicate this information"?